batcher/aligned-sdk/src/sdk.rs

```diff
@@ -129,6 +270,62 @@ async fn _submit_multiple(
     Ok(aligned_verification_data)
 }
 
+/// Submits a proof to the batcher to be verified in Aligned and waits for the verification on-chain.
+/// # Arguments
+/// * `batcher_addr` - The address of the batcher to which the proof will be submitted.
+/// * `eth_rpc_url` - The URL of the Ethereum RPC node.
+/// * `chain` - The chain on which the verification will be done.
+/// * `verification_data` - The verification data of the proof.
+/// * `wallet` - The wallet used to sign the proof.
+/// # Returns
+/// * The aligned verification data obtained when submitting the proof.
+/// # Errors
+/// * If there is an error connecting to the batcher.
+/// * If there is an error serializing the message.
+/// * If there is an error deserializing the message.
+pub async fn submit_and_wait(
+    batcher_addr: &str,
+    eth_rpc_url: &str,
+    chain: Chain,
+    verification_data: &VerificationData,
+    wallet: Wallet<SigningKey>,
+) -> Result<Option<AlignedVerificationData>, errors::SubmitError> {
+    let (ws_stream, _) = connect_async(batcher_addr)
+        .await
+        .map_err(errors::SubmitError::ConnectionError)?;
+
+    debug!("WebSocket handshake has been successfully completed");
+    let (ws_write, ws_read) = ws_stream.split();
+
+    let ws_write = Arc::new(Mutex::new(ws_write));
+
+    let verification_data = vec![verification_data.clone()];
+
+    let eth_rpc_provider = Provider::<Http>::try_from(eth_rpc_url)
+        .map_err(|e: url::ParseError| errors::SubmitError::EthError(e.to_string()))?;
+
+    let contract_address = match chain {
+        Chain::Devnet => "0x1613beB3B2C4f22Ee086B2b38C1476A3cE7f78E8",
+        Chain::Holesky => "0x58F280BeBE9B34c9939C3C39e0890C81f163B623",
+    };
+
+    let aligned_verification_data = _submit_multiple_and_wait(
+        ws_write,
+        ws_read,
+        eth_rpc_provider,
+        contract_address,
+        &verification_data,
+        wallet,
+    )
+    .await?;
```

Comment:
this could be replaced with something like 

```
submit_multiple_and_wait(...).await?;
```

Reply:
Solved in [4cdbfd5](https://github.com/yetanotherco/aligned_layer/pull/583/commits/4cdbfd5c189ff1c236950dd015206d52be74f229).